spec/Arrays.tex

```diff
@@ -382,38 +380,6 @@ \subsection{Associative Array Indexing}
 \end{chapel}
 \end{chapelexample}
 
-The second way is more concise, and has the same effect as the first method:
```

Comment:
I like the idea of removing things from the spec wholesale rather than deprecating them there, just to maximize the degree to which the spec describes our current vision of the library (at least, as much as we're able to).